viadot/sources/sap_rfc.py

```diff
@@ -100,13 +100,24 @@ class SAPRFC(Source):
     - etc.
     """
 
-    def __init__(self, sep: str = None, func: str = "RFC_READ_TABLE", *args, **kwargs):
+    def __init__(
+        self,
+        sep: str = None,
+        func: str = "RFC_READ_TABLE",
+        rfc_character_limit: int = 400,
```

Comment:
This is ambiguous as there are different SAP RFC character limits. So this should be sth like `rfc_total_col_width_character_limit`.